lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1415,5 +1421,26 @@ def api_client
       end
       @client
     end
+
+    # Encode by UTF-8 with invalid characters replaced by spaces.
+    def coerce_to_utf8_encode(input)
+      if @allow_non_utf8
+        input.encode(
+          'utf-8',
+          invalid: :replace,
+          undef: :replace,
+          replace: ' ')
+      else
+        begin
+          input.encode('utf-8')
+        rescue EncodingError
+          @log.error 'Encountered encoding issues potentially due to non ' \
+                     'UTF-8 characters. To allow non UTF-8 characters and ' \
+                     'replace them with spaces, please set "allow_non_utf8" ' \
+		     'to true.'
```

Comment:
Oops. I changed the wording last minute, and did not run the test again. Lesson learnt.